[zh-cn] sync architecture/nodes.md (#49816)
kubernetes/website: content/zh-cn/docs/concepts/architecture/nodes.md

You can modify Node objects regardless of the setting of `--register-node`.
For example, you can set labels on an existing Node or mark it unschedulable.

You can set optional node role(s) for nodes by adding one or more

> `node-role.kubernetes.io/<role>: <role>` labels to the node wherecharacters of `<role>` are limited by the

Reviewer: ```suggestion
`node-role.kubernetes.io/<role>: <role>` labels to the node where characters of `<role>` are limited by the
```

你可以修改 Node 对象（忽略 `--register-node` 设置）。
例如，你可以修改节点上的标签或并标记其为不可调度。

你可以通过在节点上添加一个或多个 `node-role.kubernetes.io/<role>: <role>` 标签，

> 来为节点设置可选的节点角色。其中，`<role>` 的字符受
> [标签键名格式](/zh-cn/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set) 规则限制。

Reviewer: ```suggestion
来为节点设置可选的节点角色。其中，`<role>`
的字符受[标签键名格式](/zh-cn/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set)规则限制。
```

* 架构设计文档中有关
  [Node](https://git.k8s.io/design-proposals-archive/architecture/architecture.md#the-kubernetes-node)
  的章节。

> * [节点自动扩缩](/docs/concepts/cluster-administration/node-autoscaling/)

Reviewer: ```suggestion
* [节点自动扩缩](/zh-cn/docs/concepts/cluster-administration/node-autoscaling/)
```
node-autoscaling.md will be localized in plan
